Sources/Valkey/Commands/ServerCommands.swift

```diff
@@ -1444,23 +1426,36 @@ public struct SHUTDOWN: ValkeyCommand {
         @usableFromInline let saveSelector: AbortSelectorSaveSelectorBlockSaveSelector?
         @usableFromInline let now: Bool
         @usableFromInline let force: Bool
-
-        @inlinable public init(saveSelector: AbortSelectorSaveSelectorBlockSaveSelector? = nil, now: Bool = false, force: Bool = false) {
+        @usableFromInline let safe: Bool
+        @usableFromInline let failover: Bool
+
+        @inlinable public init(
+            saveSelector: AbortSelectorSaveSelectorBlockSaveSelector? = nil,
+            now: Bool = false,
+            force: Bool = false,
+            safe: Bool = false,
+            failover: Bool = false
+        ) {
```

Comment:
We will need to come up with a plan so that this isn't an API break anymore, once we have tagged 1.0